components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/persistence/JDBCPersistenceManager.java

```diff
@@ -106,6 +109,21 @@ private void initDataSource() {
                 Context ctx = new InitialContext();
                 dataSource = (DataSource) ctx.lookup(dataSourceName);
             }
+            OMElement sessionPersistElem = persistenceManagerConfigElem.getFirstChildWithName(
+                    new QName(IdentityCoreConstants.IDENTITY_DEFAULT_NAMESPACE, "SessionDataPersist"));
```

Comment:
Better to have constants for "SessionDataPersist" , "DataSource" and "Name"
private static final String

Reply:
done